system/Email/Email.php

```diff
@@ -58,9 +58,15 @@ class Email
 	/**
 	 * Properties from the last successful send.
 	 *
+	 * @var array|null
+	 */
+	public $archive;
```

Comment:
How about making this variable protected and adding `getArchive()` method with a parameter that accepts `null|string` value?

Reply:
Oh... I get it. We should leave it as it is.